assets/js/blocks/handpicked-products/block.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "woocommerce/handpicked-products",
+    "version": "1.0.0",
+    "title": "Hand-picked Products",
+    "category": "woocommerce",
+    "keywords": [
+        "Handpicked Products",
+        "WooCommerce"
+    ],
+    "description": "Display a selection of hand-picked products in a grid.",
```

Comment:
here and in couple of other places in old pattern we used i18n `__` methods to provide translated strings. With `block.json` we don't have the ability to provide that here. Have you looked into how we should go about this?

Reply:
Would be nice if it was added as a translatable string automatically

Reply:
Yes, if you check and the end of the file there's this property `"textdomain": "woo-gutenberg-products-block"` which is used to load translations (https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#text-domains)

Reply:
ah, perfect!